Website/src/items/content/EdgeWebsite/home/sponsors.yml

```diff
@@ -10,84 +10,100 @@ SharedFields:
 - ID: "f1a1fe9e-a60c-4ddb-a3a0-bb5b29fe732e"
   Hint: __Renderings
   Value: |
-    <r xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <r xmlns:p="p" xmlns:s="s"
+      p:p="1">
```

Comment:
As you can see by checking the changes of this file:
1. This page was not patching the Webpage template standard values presentation details. It was overriding the template presentation details and was redefining all the renderings.
2. It still has a FeaturedVendors (18D56CF9-81D3-50C2-A8C3-19349D36074D) rendering.

The Renderings field XML must be cleaned. We have a couple options:
1. By hand. I have experience with this and can help.
2. Restart from scratch. Use the "Reset" button in the Presentation ribbon tab. Reset both the shared and final layout of that page. It will be reset to patch the Website template presentation details with no extra rendering. Then add back the appropriate renderings to the main placeholder. This is the easiest option in this case as there are not much extra renderings on this page.